tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1939,7 +1939,7 @@ def param_publish(pub, msg_class, param_name, rate, verbose):
         if rospy.is_shutdown():
             break
 
-def stdin_publish(pub, msg_class, rate, once, filename, verbose):
+def stdin_publish(pub, msg_class, rate, once, filename, verbose, substitute_keywords):
```

Comment:
```suggestion
def stdin_publish(pub, msg_class, rate, once, filename, verbose, substitute_keywords=False):
```

As this is a library downstream code could be using, the old API must be retained.